client/src/app/components/edit-profile/edit-profile.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="md-dialog.fullscreen-dialog">
+    <div class="wrapper">
+        <div class="cover-picture">
+            <img src="https://ollienollie.files.wordpress.com/2009/10/untitled-181.jpg" class="profile-pic" />
+            <button class="dot">
+                <mat-icon id="add-a-photo" aria-hidden="false">add_a_photo</mat-icon>
+            </button>
```

Comment:
Sine this button's only purpose is to host a button, the button should have the 'mat-button' attribute. Also the class name 'dot' is not very descriptive, something like 'edit-profile-picture' would be better (like you did for cover photo)